README.md

```diff
@@ -1,10 +1,260 @@
-# adminrouter
-The admin router is an open-source Nginx configuration created by Mesosphere that provides central authentication and proxy to DCOS services within the cluster.
+# Adminrouter
 
+The Adminrouter is an open-source Nginx configuration created by
```

Comment:
I will fix the readme, but if you want to be 100% consistent, we will have to rename repository as well as lots of files both in this repo and in DC/OS.